### PR TITLE
F51: Django integration

### PR DESCRIPTION
**Pull request by asaevich**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -87,3 +86,7 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+
```
Comment (ifdotpy) [inline .gitignore:89]: It should be in subfolder, not in root

File: examples/Django/emptyapi/config.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
+api_version: "0.1"
```
Comment (ifdotpy) [inline examples/Django/emptyapi/config.yaml:1]: Please, also make short form of this config. And if you skipped something here, then add field. I want to have full version and minimal version for each example.

File: internal/router/router.go [hunk 3]
```diff
@@ -16,6 +17,8 @@ var Router TRouter
 // CreateServer initializes router and creates routes for each application
 func CreateServer(apps map[string]*app.App) (*fiber.App, error) {
 	r := fiber.New()
+	// todo: REMOVE DEBUG SETTING
+	r.Use(cors.New())
```
Comment (ifdotpy) [inline internal/router/router.go:21]: In future, we should have options in config for cors. Let it be everywhere, for now. Remove comment about debug.

File: plugins/authz/jwt/handlers.go [hunk 4]
```diff
@@ -32,43 +24,73 @@ func Refresh(j *jwtAuthz) func(*fiber.Ctx) error {
...
 func getRawToken(c *fiber.Ctx, bearer bearerType, names map[string]string) (string, error) {
 	switch bearer {
 	case Header:
-		authHeader := c.Get("Authorization")
+		/*authHeader := c.Get("Authorization")
 		splitHeader := strings.Split(authHeader, "Bearer ")
 		if len(splitHeader) != 2 {
 			return "", c.Status(fiber.StatusBadRequest).JSON(&fiber.Map{
 				"success": false,
-				"message": "bearer token not in proper format",
```
Comment (ifdotpy) [inline plugins/authz/jwt/handlers.go:68]: ... is not in ...

File: examples/Django/emptyapi/core/models.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.contrib.auth.models import UserManager
+from django.db import models
+
+
+class CustomUser(AbstractBaseUser):
```
Comment (ifdotpy) [inline examples/Django/emptyapi/core/models.py:6]: Why didn't you remove it?

**Review by ifdotpy — CHANGES_REQUESTED** [review]

File: internal/configs/configs.go [hunk 6]
```diff
@@ -95,7 +95,7 @@ type (
 
 func LoadMainConfig() (*Project, error) {
 	confLoader, err := configuro.NewConfig(
-		configuro.WithLoadFromConfigFile("./config.yaml", true),
+		configuro.WithLoadFromConfigFile("./examples/Django/emptyapi/config.yaml", true),
```
Comment (ifdotpy) [inline internal/configs/configs.go:98]: WTF, bro?

**Review by ifdotpy — CHANGES_REQUESTED** [review]

File: internal/configs/configs.go (hunk 6 above)
Comment (asaevich) [reply internal/configs/configs.go:98]: For this integration i created individual config and the project should have been loaded with this config.

Comment (ifdotpy) [reply internal/configs/configs.go:98]: And you thought you should hardcode the path here?

Comment (ifdotpy) [reply internal/configs/configs.go:98]: I'll change it myself

**Pull request merged**